app/brave_generated_resources.grd

```diff
@@ -310,6 +310,15 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
       <message name="IDS_BRAVE_SYNC_V2_MIGRATE_INFOBAR_COMMAND" desc="Command text shown to users who used a previous version of Brave Sync and need to perform setup again for the new version.">
         Begin Setup
       </message>
+      <message name="IDS_BRAVE_SYNC_ACCOUNT_DELETED_INFOBAR_MESSAGE" desc="Message text shown to users who's Sync account was deleted. In continuation goes the link IDS_BRAVE_SYNC_ACCOUNT_DELETED_INFOBAR_LINK_TEXT">
+        This Brave Sync account was deleted from other device. If you didn’t mean to remove the Sync,
+      </message>
+      <message name="IDS_BRAVE_SYNC_ACCOUNT_DELETED_INFOBAR_BUTTON" desc="Button text shown to users who's Sync account was disabled. This button prevents further show of the infobar.">
+        OK
+      </message>
+      <message name="IDS_BRAVE_SYNC_ACCOUNT_DELETED_INFOBAR_LINK_TEXT" desc="LInk text shown to users who's Sync account was deleted. When user clicks the link, Sync Setup page is opened">
```

Comment:
typo: `LInk` instead of `Link` in the description.

Reply:
thanks, fixed at 
https://github.com/brave/brave-core/pull/13562/commits/633b9fd8629179bde34878b1f3d338a1d43c94bf